src/battle-animations.ts

```diff
@@ -1872,7 +1874,9 @@ class PokemonSprite extends Sprite {
 			statbarOffset += this.isBackSprite ? 1 : 20;
 		} else if (this.scene.gen <= 3) {
 			statbarOffset += this.isBackSprite ? 5 : 30;
-		} else {
+		// TODO: is this mechanicsGen? graphicsGen? spriteGen?
+		// TODO: if we're scaling gen5 backsprites still we likely still need some offset?
+		} else if (this.scene.gen !== 5) {
```

Comment:
@Zarel

Reply:
`scene.gen` is mechanics gen. We don't technically need it since we _can_ just read `scene.battle.gen`, I suppose.

Reply:
lol, it turns out I'm wrong:

https://github.com/Zarel/Pokemon-Showdown-Client/blob/21fa46b9a60079902fbe8080a52ddd61a98e6f67/src/battle-animations.ts#L38-L39

We should probably rename it to `graphicsGen`.

Reply:
I'm punting on this renaming for now because it actually is used in a lot of places and I dont want to cause breakages. I'd like to ship as is and fix in my followup adding the noScale and alternative gen support.